File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let result = [];
    let duplicate = [];
    nums.forEach(n => {
        !result.includes(n) && result.push(n)
    })
    for(let i = 0; i<result.length; i++){
        nums[i] = result[i]
    }
    for(let i = result.length; i < nums.length; i++) {
        nums[i] = "_"
    }
    return result.length;
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.go
func removeDuplicates(nums []int) int {
    dup := 0
    unique := 1
    for i:=0; i< len(nums); i++ {
        if i == 0 {
            dup = nums[0]
        } else {
            if nums[i] != dup{
                nums[unique] = nums[i]
                dup = nums[i]
                unique++
            } 
        }
    }
    return unique
}